Gandalan.IDAS.WebApi.Client/Tiles/ITileItem.cs

```diff
@@ -11,8 +10,9 @@ public interface ITileItem
 		string Name { get; set; }
 		object Control { get; set; }
         string Group { get; set; }
-        Color StateColor { get; set; }
+        object StateColor { get; set; }
 		int Order { get; set; }
+		bool IsVisible { get; set; }
 
 		Task Load();
```

Comment:
Fix indentation for `ITitleItem.cs` file

Reply:
Done